app/assets/javascripts/discourse/tests/acceptance/sticky-avatars-test.js

```diff
@@ -0,0 +1,35 @@
+import { acceptance, query } from "discourse/tests/helpers/qunit-helpers";
+import { test } from "qunit";
+import { settled, visit } from "@ember/test-helpers";
+import { later } from "@ember/runloop";
+
+acceptance("Sticky Avatars", function (needs) {
+  const container = document.getElementById("ember-testing-container");
+
+  needs.hooks.beforeEach(function () {
+    container.scrollTop = 0;
+  });
+
+  test("Adds sticky avatars when scrolling up", async function (assert) {
+    const appEvents = this.owner.lookup("service:app-events");
+
+    await visit("/t/internationalization-localization/280");
+
+    container.scrollTo(0, 800);
+    appEvents.trigger("topic:scrolled", 800);
```

Comment:
This event isn’t triggered automatically as it should be so we need to manually trigger it here to make the test pass. I guess it’s related somehow with the fact we’re observing `window` for scroll events instead of the ember container (not sure if this is fixable at some point).

Reply:
@eviltrout note that we note you didn't recommend using appEvents in test, but we couldn't figure another way and also it's already extensively used in uppy tests.

Reply:
I guess that’s because the topic scroll logic is bound somehow to `window` and here it’s the ember container that is scrolling. (We didn’t dig too deep though)

Reply:
OK cool. Long term I would like to remove it but we can live with it for a bit.

Reply:
Actually just found a better way that works every time locally both on FF and Chrome. No need for `appEvents` anymore :tada: 
I just hope this is going to work on the CI too :sweat_smile: